lib/rex/ui/text/shell/history_manager.rb

```diff
@@ -1,4 +1,5 @@
 # -*- coding: binary -*-
+require 'readline'
```

Comment:
I'm not sure if this is the right fix based on https://github.com/rapid7/metasploit-framework/blob/fd43c43a30e42e02058bd03455d37fff7e7342b0/lib/msf/ui/console/driver.rb#L658

Potentially the readline require belongs elsewhere, i.e. in the tool itself

Reply:
Yeah that's probably a safer approach. I'll switch it over.